src/main/java/frc/robot/controls/Operator.java

```diff
@@ -95,14 +93,22 @@ public Operator(Robot robot)
         climberUnwind.whenHeld(new ClimberUnwind(robot));
         climberExtend.whenHeld(new ClimberExtend(robot));
         climberRetract.whenHeld(new ClimberRetract(robot));
-
-        shooterSpin.whenHeld(new ShooterSpin(robot));
```

Comment:
Isn't the default command ShooterStop?  So when you press the button, it switches to running ShooterSpin, and when you stop pressing it we go back to ShooterStop.

Also I realized that ShooterSpin is returning `false` (running forever) and ShooterStop is returning `true` but it should be the other way around right?  @LegenWaitForIt-Dary what do you think?

Reply:
You're right, the default command is ShooterStop, I read it wrong. I'll change that.
 
Currently the button is just enabling the joystick, so to me it makes more sense to get rid of the button and just have the shooter constantly reading from the joystick. If we do that, then ShooterSpin should return `false` since it's always running and it doesn't matter what ShooterStop returns because its not being used.

Reply:
Are you manually setting the shooter speed with the joystick? Is this just for testing? Since I doubt the operator will be able to accurately move the joystick to the perfect speed every time. Ideally, you would want just a shoot button to press and the speed is automatically calculated based on vision. I think that was what @bricktrooper was referring to when he meant pressing the button to shoot.

Reply:
Yeah the idea is that the operator can vary the speed of the motor using the controller.  But when vision takes control of the shooter, it will switch to the calculated speed instead.

Reply:
> You're right, the default command is ShooterStop, I read it wrong. I'll change that.
> 
> Currently the button is just enabling the joystick, so to me it makes more sense to get rid of the button and just have the shooter constantly reading from the joystick. If we do that, then ShooterSpin should return `false` since it's always running and it doesn't matter what ShooterStop returns because its not being used.

Hmm yeah @Geotricity you're right this makes more sense.  So spin is always running in the background like the Drive command.  Therefore we should make ShooterSpin the default (return false) and ShooterStop return true.  DOes this make sense to everyone?

Reply:
> Are you manually setting the shooter speed with the joystick? Is this just for testing? Since I doubt the operator will be able to accurately move the joystick to the perfect speed every time. Ideally, you would want just a shoot button to press and the speed is automatically calculated based on vision. I think that was what @bricktrooper was referring to when he meant pressing the button to shoot.

Like @bricktrooper said, both this and automatic speed from the vision will be be available to the operator. I'm working on getting the vision speed setting working before this Saturday.